### ISSUE TITLE
[request] distribute a statically linked binary

### ISSUE THREAD

**Issue by skaji** [motemen/ghq#267]
Currently https://github.com/motemen/ghq/releases/download/v1.0.1/ghq_linux_amd64.zip contains a dynamically linked binary, so we cannot use it in alpine linux:
```
❯ docker run -it alpine sh -c 'set -x; apk add file; wget https://github.com/motemen/ghq/releases/download/v1.0.1/ghq_linux_amd64.zip; unzip ghq_linux_amd64.zip; file ./ghq_linux_amd64/ghq; ldd ./ghq_linux_amd64/ghq; ./ghq_linux_amd64/ghq --version'
+ apk add file
fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/community/x86_64/APKINDEX.tar.gz
(1/2) Installing libmagic (5.36-r1)
(2/2) Installing file (5.36-r1)
Executing busybox-1.29.3-r10.trigger
OK: 11 MiB in 16 packages
+ wget https://github.com/motemen/ghq/releases/download/v1.0.1/ghq_linux_amd64.zip
Connecting to github.com (13.114.40.48:443)
Connecting to github-production-release-asset-2e65be.s3.amazonaws.com (52.216.24.84:443)
ghq_linux_amd64.zip  100% |*******************************************************| 3347k  0:00:00 ETA
+ unzip ghq_linux_amd64.zip
Archive:  ghq_linux_amd64.zip
   creating: ghq_linux_amd64/
  inflating: ghq_linux_amd64/CHANGELOG.md
  inflating: ghq_linux_amd64/CREDITS
  inflating: ghq_linux_amd64/LICENSE
  inflating: ghq_linux_amd64/README.adoc
  inflating: ghq_linux_amd64/ghq
   creating: ghq_linux_amd64/misc/
   creating: ghq_linux_amd64/misc/zsh/
  inflating: ghq_linux_amd64/misc/zsh/_ghq
+ file ./ghq_linux_amd64/ghq
./ghq_linux_amd64/ghq: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, Go BuildID=JDrKCI80mqsR2R5EnwGt/HSGlt0h2MDX8mDtfQbmu/vUhlEgzeJ0n9BfujgOgM/l8wlgdb36Brl_Exh-uBD, stripped
+ ldd ./ghq_linux_amd64/ghq
	/lib64/ld-linux-x86-64.so.2 (0x7f913b6f2000)
	libpthread.so.0 => /lib64/ld-linux-x86-64.so.2 (0x7f913b6f2000)
	libc.so.6 => /lib64/ld-linux-x86-64.so.2 (0x7f913b6f2000)
+ ./ghq_linux_amd64/ghq --version
sh: ./ghq_linux_amd64/ghq: not found
```

I think it would be nice if ghq is distributed as a statically linked binary.

**Comment by Songmu** [comment]
It may be resolved at v1.0.2. Please try it.

**Comment by skaji** [comment]
Now it works.
```
❯ docker run -it alpine sh -c 'set -x; apk add file; wget https://github.com/motemen/ghq/releases/download/v1.0.2/ghq_linux_amd64.zip; unzip ghq_linux_amd64.zip; file ./ghq_linux_amd64/ghq; ldd ./ghq_linux_amd64/ghq; ./ghq_linux_amd64/ghq --version'
+ apk add file
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz
(1/2) Installing libmagic (5.37-r1)
(2/2) Installing file (5.37-r1)
Executing busybox-1.31.1-r8.trigger
OK: 11 MiB in 16 packages
+ wget https://github.com/motemen/ghq/releases/download/v1.0.2/ghq_linux_amd64.zip
Connecting to github.com (52.192.72.89:443)
Connecting to github-production-release-asset-2e65be.s3.amazonaws.com (52.217.43.132:443)
saving to 'ghq_linux_amd64.zip'
ghq_linux_amd64.zip  100% |*******************************************************| 3323k  0:00:00 ETA
'ghq_linux_amd64.zip' saved
+ unzip ghq_linux_amd64.zip
Archive:  ghq_linux_amd64.zip
   creating: ghq_linux_amd64/
  inflating: ghq_linux_amd64/CHANGELOG.md
  inflating: ghq_linux_amd64/CREDITS
  inflating: ghq_linux_amd64/LICENSE
  inflating: ghq_linux_amd64/README.adoc
  inflating: ghq_linux_amd64/ghq
   creating: ghq_linux_amd64/misc/
   creating: ghq_linux_amd64/misc/zsh/
  inflating: ghq_linux_amd64/misc/zsh/_ghq
+ file ./ghq_linux_amd64/ghq
./ghq_linux_amd64/ghq: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=EfTui6NNL_tXI9sl9Z-q/zgHwLab4gK6o5xfEoJus/f3ywAbpjiuqpH9ZvIuNx/N5e4sag1YNdhl-IqZenZ, stripped
+ ldd ./ghq_linux_amd64/ghq
/lib/ld-musl-x86_64.so.1: ./ghq_linux_amd64/ghq: Not a valid dynamic program
+ ./ghq_linux_amd64/ghq --version
ghq version 1.0.2 (rev:3981090)
```

@Songmu Thank you!
